admin/css/picker.css

```diff
@@ -4,18 +4,23 @@
 :root {
   --sidebar-width-large: 450px;
   --sidebar-width-tablet: 400px;
+  --button-color: #3858e9;
```

Comment:
I know this is part of the designs, but I personally think it looks weird, WordPress buttons in the rest of the interface look different (this color is for the Gutenberg editor only if I'm not mistaken)

Reply:
Did you ask Taylor about this @GP-Dan-Tovbein ?